pkg/kubecfg/fixtures/kubeconfig.yml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+clusters:
+- cluster:
+    certificate-authority-data: XXXX
+    server: https://api.live-1.cloud-platform.service.justice.gov.uk
+  name: mogaal.cloud-platform.service.justice.gov.uk
+contexts:
+- context:
+    cluster: mogaal.cloud-platform.service.justice.gov.uk
+    user: alejandro.garrido@digital.justice.gov.uk
+  name: mogaal.cloud-platform.service.justice.gov.uk
+current-context: ""
+kind: Config
+preferences: {}
+users:
+- name: alejandro.garrido@digital.justice.gov.uk
+  user:
+    auth-provider:
+      config:
+        client-id: m1yfMHQDYSxncbhmUNIrm1yfMHQDYSxn
+        client-secret: 50fk9Tlc1ESIfUE4CKSCEUo3mYCKfw7mNVKmOcg1fMAnQEMFyLaKF2Kie8XG
```

Comment:
Am I right in thinking that these are real-looking but not actually real client-secret and id-token values?

Reply:
They are not real, just generated a random string [here](http://www.unit-conversion.info/texttools/random-string-generator/)